ui/src/components/QRDeposit.tsx

```
@@ -226,34 +232,136 @@ const QRDeposit: React.FC = () => {
         }
     }, [fetchWeb3Balance, web3Address]);
 
+    // Handle form submission for Bitcoin payments.
+    const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+        e.preventDefault();
+
+        if (BITCOIN_PAYMENTS) {
+            const formData = new FormData(e.currentTarget);
+
+            debugger;
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)